gctscript/wrappers/gct/exchange/exchange.go

```diff
@@ -88,7 +88,7 @@ func (e Exchange) QueryOrder(exch, orderID string) (*order.Detail, error) {
 		return nil, err
 	}
 
-	r, err := ex.GetOrderInfo(orderID)
+	r, err := ex.GetOrderInfo(orderID, currency.Pair{}, "")
```

Comment:
QueryOrder needs to have assetType and pair too so they can be used in GetOrderInfo

Reply:
+